kansha/authentication/ldap/forms.py

```diff
@@ -20,18 +20,18 @@ class Login(Authentication):
 
     alt_title = None
 
-    config_spec = {
+    CONFIG_SPEC = {
         'activated': 'boolean(default=False)',
         'server': 'string(default="")',
         'user_base_dn': 'string(default="")',
         'cls': 'string(default="")'
```

Comment:
Never set 'default' values that will crash:
  - don't set any 'default' to force the user to set a value (i.e 'server': 'string')
  - or set some "generic" values (i.e 'server': 'string(default="localhost")')